src/styles/blocks/_about.scss

```diff
@@ -0,0 +1,75 @@
+.about {
+
+  padding: 18.75vw 6.25vw;
+
+  text-align: left;
+
+  display: flex;
+
+  flex-direction: column;
+  gap: 10vw;
+
+  &__title {
+    letter-spacing: -1.5px;
+    font-weight: 500;
+
+    font-size: 32px;
+    line-height: 32px;
+  }
+
+  &__content {
+    font-weight: 500;
+
+    font-size: 23px;
+    line-height: 32px;
+  }
+
+  height: 127.5vw;
+
+  @media (min-width: 480px) {
+
+    height: 59.68vw;
+
+    padding: 16.13vw 9.68vw;
+
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 3.23vw;
+
+    &__title {
+      font-size: 48px;
+      line-height: 100%;
+      width: 38.71vw;
+    }
+
+    &__content {
+      font-size: 23px;
+      line-height: 140%;
+      width: 38.71vw;
+    }
+  }
+
+  @media (min-width: 1024px) {
+
+    height: 30.16vw;
+
+    padding: 9.52vw 16.43vw 13vw 9.52vw;
+
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 1.9vw;
+
+    &__title {
+      font-size: 64px;
+      line-height: 100%;
+      width: 32.62vw;
+    }
+
+    &__content {
+      font-size: 23px;
+      line-height: 140%;
+      width: 39.5vw;
+    }
```

Comment:
Why do you repeat styles? We are using Sass, you can nest media queries like this:
```scss

.block {
  color: red;
  font-size: 12px;

  @media (min-width: 640px) {
    color: blue;
  }

  @media (min-width: 1280px) {
    color: yellow;
    font-size: 16px;
  }
}
```

Also, create mixins for mobile, tablet and desktop queries.
```scss
@mixin onTablet {
  @media (min-width: $tabletBreakpoint) {
    @content;
  }
}
```